EIPS/eip-5585.md

```diff
@@ -0,0 +1,111 @@
+
+## Motivation
+
+The development of NFT has simplified the ownership verification and transaction. However, other kinds of rights except ownership such as exhibition rights, lease rights, creation of derivatives etc. are still difficult to be circulated and used independently, which have limited the value and commercialization of digital assets.
+
+As the development of the Metaverse, digital assets are diverse on the blockchain such as digital collections, virtual real estate, and commercial copyrights etc.,  the commercial rights (exhibition rights, derivative rights, commercial authorization, etc.) based on digital assets are becoming a potential business form. 
+
+After separating the ownership of assets and other commercial rights, other problems will be raised. For example, when the ownership changes, at the same time the commercial authorization is still valid, it will result the conflict interest between the new owner and the commercial authorization user. It is extremely important thing how to protect the interests of both parties.
+
+## Specification
+
+```solidity
+struct UserRecord {
+address user;
+ uint expires
+}
+
+// The total users of every single NFT can be authorized to set by creators
+uint256 userLimit = 5;
+
+//The starting time of staking
+mapping(uint256 => uint) timeOfStaking;
+
+//if resetAllowed == false£¬the authorization can be revoked; otherwise, it cannot be revoked. 
+bool resetAllowed = false;
+
+// Mapping tokenID to user amount
+mapping(uint256 => uint256) private _userAmount;
+
+//Store the authorized users and their expire time
+mappint(uint256 => mapping (uint => UserRecord)) expires;
+
+//Store the index of users authorization
+mappint(uint256 => mapping (address => uint)) _userTokensIndex;
+
+//NFT creators can set the starting time of the staking
+function stakingTimeUpdate(address[] user, uint[] stakingTime) external;
+
+//Update userLimit and resetAllowed
+function userLimitAndresetAllowedUpdate(uint256 limit, bool allowed) external onlyOwner;
+
+/**
+ * @dev Authorize the usage of the token(the owner or approver).
+ * @param tokenId The token ID.
+ * @param user The address that should use the token.
+ * @param duration Durations in seconds for the authorization.
+*/
+
+// NFT Owner authorize NFT to users.
+function userAuthorization(uint256 tokenId, address user, uint duration) external returns(uint) 
+
+// NFT Owner extends the duration of authorization 
+function durationUpdate(uint256 tokenId, address user, uint duration) external returns(uint)
+
+// Get the times of expired for one specific tokenId and user.
+function userExpiresTime(uint256 tokenId, address user) external view returns(uint) 
+
+// Check if the token is available for authorization.
+function authorizationAvailabilityCheck (uint256 tokenId) public view returns(bool) 
```

Comment:
These are not descriptive enough. Describe what each parameter does and what the return value is.